script/Ciel/Day 5/05_01_CIEL05_3B.txt

```diff
@@ -86,78 +94,85 @@ I think I have the right to ask her who exactly she is.
 [sha:9e981e5d81d9e3c1280ed7c15d948e1993d4578e]{
 -- Page 444, Offset 12043.
 -- ……でも、何を言えばいいのか分からなかった。
-...But, I don't know what I should say.
+But... I don't know exactly what I should say.
 }
 [sha:a556a7a50f0b1895ac0f661da12456f2c9fc15f1]{
 -- Page 445, Offset 12044.
 -- 先輩は何者なんですか、とか、
-Something like, 'Who exactly are you?'
+'Just who are you?'
 }
 [sha:80e0dbdb60682a40ca23aa5d45fdfe5d1c97552c]{
 -- Page 445, Offset 12045.
 -- 怪我はなかったんですか、とか、
-Or, 'Are you injured?'
+'Are you injured?'
 }
 [sha:987abaa80baeffc9de4a042381a4bebd30b621a6]{
 -- Page 445, Offset 12046.
 -- これからどうするんですか、とか、
-Or, 'What are you going to do from now on?'
+'What are you going to do now?'
 }
 [sha:a84bb67564002289041a3d33448e8862d4ae6983]{
 -- Page 445, Offset 12047.
 -- 先生とは何もなかったです、とか。
-Or, 'Nothing happened between me and Sensei'
+'Nothing happened between me and Sensei.'
 }
 [sha:1fbef179f0d2011e8746c7080293cc6dfc38be0e]{
 -- Page 445, Offset 12048.
 -- そんなありきたりの言葉が、どうしても口からでない。
-I couldn't speak out any of these common phrases.
+No matter how hard I try, I can't bring myself to say any of those obvious phrases.
 }
 [sha:d133ff8edb6fcc2e9cd61d64c84267fd9effecc7]{
 -- Page 446, Offset 12049.
 -- 空を見れば、もう夕暮れも終わりかけている。
-Looking at the sky, dusk is coming to an end.
+Judging by the sky, dusk is coming to an end.
 }
 [sha:189ad3fcf33e258a0e603e9feff0ef4b68a8828b]{
 -- Page 446, Offset 12050.
 -- 日没まであとわずか。
-The sun will completely set very soon.
+Not long now before sunset.
```

Comment:
since [dusk comes after sunset](https://en.wikipedia.org/wiki/Dusk), I'm guessing this is actually "not long until nightfall" as in "sunset will be over before long" though it is quite a stretch to interpret it that way

Reply:
I'll change 夕暮れ to evening instead, that word is a bit easier to weasel.